### DISCUSSION TITLE
请问一下这个报错怎么解决

### DISCUSSION THREAD

**Discussion by Annnnh1010** [nonebot/discussions#355] [问答] [answered]
![VeryCapture_20230322192132](https://user-images.githubusercontent.com/76246787/226889500-f4c5b350-98f4-4b7a-a0b7-ec6888df43ca.jpg)

**Comment by yanyongyu** [comment]
你的driver配置写错了

**Reply by Annnnh1010** [reply]
应该没错啊。

**Reply by yanyongyu** [reply]
你截个图看看

**Reply by Annnnh1010** [reply]
![VeryCapture_20230323173604](https://user-images.githubusercontent.com/76246787/227162439-b12d109a-4017-45c6-a213-62d529956f34.jpg)
![VeryCapture_20230323173627](https://user-images.githubusercontent.com/76246787/227162445-be4000d1-d63f-4884-bedb-36ffb52e1d06.jpg)

**Comment by yanyongyu** [comment]
你这driver写的啥玩意儿啊。。。。

```
DRIVER=~websockets
```

**Reply by Annnnh1010** [reply]
我明白你的意思了 但是又出现了新的问题
![VeryCapture_20230323182716](https://user-images.githubusercontent.com/76246787/227175661-95461ea0-ff01-4154-97d5-f311facd88c1.jpg)

**Comment by Annnnh1010** [comment, marked as answer]
![VeryCapture_20230323183141](https://user-images.githubusercontent.com/76246787/227176556-e1319c7c-8cb8-4b9b-b04d-908ef1976165.jpg)

**Reply by yanyongyu** [reply]
```
DRIVER=~fastapi+~httpx+~websockets
```

**Reply by Annnnh1010** [reply]
感谢大佬解答
